assignment-6/src/App.js

```diff
@@ -0,0 +1,77 @@
+import { useEffect, useState } from 'react';
+import './styles.css';
+import { CardComponent, TitleComponent, SearchBar } from './Components';
+
+const CardContainer = ({ userData }) => {
+  return (
+    <div className='container'>
+      <div className='row'>
+        {
+          userData.map((singleUser) => {
+            // console.log(singleUser);
+            return <CardComponent singleUser={singleUser} key={singleUser.id}/>
+          })
+        }
+      </div>
+    </div>
+  )
+}
+
+const usernames = [
+  "akshaymarch7", 
+  "ap221882", 
+  "nikitaj-57", 
+  "Pujarini", 
+  "aditifarkya",
+  "gavandivya",
+  "soumyagangamwar",
+  "Bhallora"
+]
+
+const BodyComponent = () => {
+  // UserData will store the data fetched using Github API
+  const [ userData, setUserData ] = useState([]);
+
+  const fetchUserDetails = async() => {
```

Comment:
API fetch call can be separated into a separate file. As the project grows, there will be multiple API calls, and you should make this code reusable.

Reply:
I think that is okay for now. Since this is one page. also we should keep api calls specific to the module and not make it generic unless required by all of the whole application like login details.